src/mutations/user/walletLogin.ts

```diff
@@ -232,6 +233,18 @@ const _walletLogin: Exclude<
       })
       await userService.postRegister(user)
     }
+
+    // no await to put data async
+    atomService.aws.putMetricData({
+      MetricData: [
+        {
+          MetricName: METRICS_NAMES.UserRegistrationCount,
+          Timestamp: new Date(),
+          Unit: 'Count',
+          Value: 1,
+        },
+      ],
+    })
```

Comment:
and socialLogin?

Reply:
revised to handle metrics emission in `userService` better